### ISSUE TITLE
Audio Capture plugin crashes on enable on windows

### ISSUE THREAD

**Issue by pfaion** [pupil-labs/pupil#1560]
I saw that there are some other issues with audio capture crashing, but mine does not even start. I started capture and enabled the **Audio Capture** plugin. Then capture crashed immediately.

More details:
- I am building from source on current master (1a8ae879e5757fa3c719a1159ee1cb91bfdf57c5)
- Running windows 10
- I have no microphone, but a headset registered as sound input (tested, working)
- Python 3.6
- pyaudio==0.2.11
- In capture I had the **Local USB** backend enabled and the Pupil Core headset connected. All videos (world + 2*eye) were already available.

This is the error message:

```
world - [ERROR] launchables.world: Process Capture crashed with trace:
Traceback (most recent call last):
  File "C:\Users\PFA\Documents\git\pupil/pupil_src\launchables\world.py", line 611, in world
    handle_notifications(n)
  File "C:\Users\PFA\Documents\git\pupil/pupil_src\launchables\world.py", line 403, in handle_notifications
    g_pool.plugin_by_name[noti["name"]], args=noti.get("args", {})
  File "C:\Users\PFA\Documents\git\pupil\pupil_src\shared_modules\plugin.py", line 352, in add
    plugin_instance = new_plugin(self.g_pool, **args)
  File "C:\Users\PFA\Documents\git\pupil\pupil_src\shared_modules\audio_capture.py", line 45, in __init__
    self.audio_devices_dict = Audio_Input_Dict()
  File "C:\Users\PFA\Documents\git\pupil\pupil_src\shared_modules\audio\__init__.py", line 167, in __init__
    for dev_info in get_input_devices_by_api(pa.paDirectSound):
  File "C:\Users\PFA\Documents\git\pupil\pupil_src\shared_modules\audio\__init__.py", line 44, in get_input_devices_by_api
    ds_info = pyaudio.get_host_api_info_by_type(api)
  File "C:\Python\Python36\lib\site-packages\pyaudio.py", line 816, in get_host_api_info_by_type
    index = pa.host_api_type_id_to_host_api_index(host_api_type)
OSError: [Errno -9979] Host API not found
```

**Comment by pfaion** [comment]
Need to check if the issue exists also on other platforms!

**Comment by pfaion** [comment]
I tested on the last release bundle (v1.13) for windows.
Here the issue does NOT occur!

**Comment by pfaion** [comment]
I think the issue was caused by a corrupt installation of pyaudio on my machine, as all bundles worked, while my build from source did not. I managed to workaround the issue by copying the `_portaudio.cp36-win-amd64.pyd` from the bundle into my local site-packages folder. Now it works.

A better way might have been to compile portaudio and pyaudio from source though, but noting this here as a quick-fix-reference.
